@uportal/user-profile-menu/src/components/ProfileMenu.vue

```diff
@@ -172,6 +172,9 @@ export default {
   }
 
   .profile-dropdown-section {
+    &:empty {
+      display: none;
+    }
```

Comment:
:art: style recommendation from linter

```suggestion
    }

```